app/src/main/java/org/piramalswasthya/cho/ui/home/HomeFragment.kt

```
@@ -193,6 +199,20 @@ class HomeFragment : Fragment() {
             searchPrompt.show()
 
         }
+        binding.report.setOnClickListener {
+            try {
+                val url = getString(R.string.url_dhis_report) + preferenceDao.getDhisToken()
+                val intent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
+
+                startActivity(intent)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)